### PR TITLE
Dual Phone Hold Test

### PR DESCRIPTION
**Pull request by dephillipsmichael**
The JSON wasn't very complicated, but supporting the whichHand and nextHand functionality with different section step identifiers was confusing.  I tested each scenario and all of them are the correct flow.

### CODE REVIEW

File: MotorControl/MotorControl/Resources/RestingKineticTremor.json [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+    "identifier":"RestingKineticTremor",
+    "usesTrackedData":true,
+    "shouldHideActions":["goBackward", "skip"],
+    "progressMarkers":[],
+    "steps":[
+        {
+            "identifier":"introduction",
+            "type":"overview",
+            "title":"Dual Phone Hold",
+            "text":"In this activity you will be holding the phone in your lap for 30 seconds. You will do this with each hand.  Goal is to measure your tremors in your hand. Then you’ll hold your phone with your pointer finger sticking out sideways. Then touch the tip of your nose with the tip of your finger. Your goal is to be as accurate as possible in touching your nose.",
```
Comment (DanWebster) [inline MotorControl/MotorControl/Resources/RestingKineticTremor.json]:
```suggestion
            "text":"In this challenge you will be holding the phone while doing 2 activities. The first is to keep your hand still in your lap, the second is to move your arm to touch the tip of your nose with the tip of your finger as accurately as you can. Please perform this activity with both hands if you can.",
```

File: MotorControl/MotorControl/Resources/RestingKineticTremor.json [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+                 "type":"fetchable",
+                 "imageName":"Completion-Tremor",
+                 "placementType":"topBackground"
+             },
+             "viewTheme":{
+                 "viewIdentifier":"Completion",
+                 "storyboardIdentifier":"ActiveTaskSteps"
+             },
+             "spokenInstructions":{
+                 "start":"Great job, you are done with the Tremor test."
```
Comment (DanWebster) [inline MotorControl/MotorControl/Resources/RestingKineticTremor.json]:
```suggestion
                 "start":"Great job, you are done with the first part of the Dual Phone Hold test."
```

File: MotorControl/MotorControl/Resources/RestingKineticTremorFirstAttemptjson.json [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+                                        "value": "both"
+                                    }
+                                ]
+                            }]
+             },
+             {
+                "identifier":"holdPhoneInstructions",
+                "type":"instruction",
+                "title":"Hold the phone still",
+                "text":"While sitting down with your feet resting flat on the floor, you will be holding the phone still in your lap for 30 seconds. You will do this first with one hand and then with your other hand.",
```
Comment (DanWebster) [inline MotorControl/MotorControl/Resources/RestingKineticTremorFirstAttemptjson.json:68]:
```suggestion
                "text":"While sitting down with your feet resting flat on the floor, please hold the phone still in your lap for 30 seconds. You will do this first with one hand and then with your other hand if you can.",
```

File: MotorControl/MotorControl/Resources/RestingKineticTremorFirstAttemptjson.json [hunk 4]
```diff
@@ -0,0 +1,179 @@
...
+                     "type":"fetchable",
+                     "imageName":"Completion-Tremor",
+                     "placementType":"topBackground"
+                 },
+                 "viewTheme":{
+                     "viewIdentifier":"Completion",
+                     "storyboardIdentifier":"ActiveTaskSteps"
+                 },
+                 "spokenInstructions":{
+                     "start":"Great job, you are done with the Tremor test."
```
Comment (DanWebster) [inline MotorControl/MotorControl/Resources/RestingKineticTremorFirstAttemptjson.json:169]:
```suggestion
                     "start":"Great job, you are done with the second part of the Dual Phone Hold test."
```

**Review by DanWebster — CHANGES_REQUESTED** [review]
Looks great, few copy changes

File: MotorControl/MotorControl/Resources/RestingKineticTremor.json (hunk 2 above)
Comment (dephillipsmichael) [reply MotorControl/MotorControl/Resources/RestingKineticTremor.json]: This is actually only shown once at the end, so I'm going to make it "Great job, you are done with the Dual Phone Hold test."

File: MotorControl/MotorControl/Resources/RestingKineticTremorFirstAttemptjson.json (hunk 4 above)
Comment (dephillipsmichael) [reply MotorControl/MotorControl/Resources/RestingKineticTremorFirstAttemptjson.json:169]: Oh I see the issue here.  That JSON was actually a debugging artifact and not used and is now deleted.  The RestingKineticTremor.json contains both tremor and kinetic tremor.

**Comment by dephillipsmichael** [comment]
@DanWebster changes made

**Comment by dephillipsmichael** [comment]
Mergining this in.  I fixed the issues and I need to push up the PKU PR targeting this.

**Pull request merged**